kitsune/sumo/static/sumo/scss/form-step.styles.scss

```diff
@@ -16,3 +18,42 @@
   line-height: 1.14;
   color: var(--color-text-form-step);
 }
+
+.hidden {
+  visibility: hidden;
+}
+
+.error-tooltip:not(.hidden) + input:invalid,
+input:not(:focus):not(:placeholder-shown):invalid {
+  border: 2px solid p.$color-red-60;
+}
+
+.error-tooltip:not(.hidden) + input:focus:invalid {
+  box-shadow: 0 0 0 2px rgba(226, 40, 80, 0.5);
+}
+
+.tooltip-container {
+  position: relative;
+}
+
+.error-tooltip {
+  background-color: p.$color-red-60;
+  border-radius: 4px;
+  color: var(--color-white);
+  padding: 8px;
+  position: absolute;
+  top: -34px;
+}
+
+.error-tooltip::before {
+  background-color: p.$color-red-60;
+  bottom: -8px;
+  content: ".";
```

Comment:
Does this actually need a character here? I think `content: ""` should do - although if this is being cribbed from the Protocol styles, maybe best to just keep what they have.

Reply:
Good catch! This was largely cribbed from FxA styles, but I've already been tweaking it a bit to fit our case.